app/components/pull-request-page.js

```diff
@@ -22,8 +31,11 @@ export default Ember.Component.extend({
     Ember.run.later(
       this,
       function() {
+        get(this, "model.status") === "draft" ? set(this, "triedToSubmit", false) : set(this, "triedToSubmit", true)
```

Comment:
I thought tried to submit was happening after the person clicks submit for the first time. Here, it's in the autoSave function. Unless I'm misunderstanding, I think it needs to move.